data/hooks/conf_regen/19-postfix

```diff
@@ -24,7 +24,17 @@ do_pre_regen() {
 
   # Support different strategy for security configurations
   export compatibility="$(yunohost settings get 'security.postfix.compatibility')"
-
+  
+  # Add possibility to specify a relay
+  # Could be useful with some isp with no 25 port open or more complex setup
+  export relay_host="$(yunohost settings get 'smtp.relay.host')"
+  if [ ! -z "${relay_host}" ]; then
+    export relay_port="$(yunohost settings get 'smtp.relay.port')"
+    export relay_user="$(yunohost settings get 'smtp.relay.user')"
+    relay_password="$(yunohost settings get 'smtp.relay.password')"
+    echo "[${relay_host}]:${relay_port} ${relay_user}:${relay_password}" > /etc/postfix/sasl_passwd
+    postmap /etc/postfix/sasl_passwd
```

Comment:
Ugh we really aren't supposed to edit the actual filesystem during the regenconf ... this should happen in `$pending_dir` (or using the `$postfix_dir` shortcut).

Also ideally we need to remove that file if the `relay_host` setting is removed later (I believe that's done by creating an empty file, c.f. the nginx regenconf)